src/pybind/mgr/dashboard/frontend/src/app/shared/api/smb.service.ts

```diff
@@ -0,0 +1,15 @@
+import { HttpClient } from '@angular/common/http';
+import { Injectable } from '@angular/core';
+
+@Injectable({
+  providedIn: 'root'
+})
+export class SmbService {
+  baseURL = 'api/smb/cluster';
+
+  constructor(private http: HttpClient) {}
+
+  list() {
+    return this.http.get(`${this.baseURL}`);
+  }
```

Comment:
also let's properly type the return type so its more readable in the future.
```suggestion
  list(): Observable<SMBCluster[]> {
    return this.http.get<SMBCluster[]>(`${this.baseURL}`);
  }
```